File: java_work/ex0701/src/ex0701/EX07.java
package ex0701;
public class EX07 {

	public static void main(String[] args) {
		doA();
		int result = doB();
		System.out.println("result = "+result);
		double dd = doC();
		System.out.println("dd = "+ dd);
		String ee = doD();
		System.out.println("ee ="+ee);
	}
	
	
	public static void doA() {
		System.out.println("doA");
		return; // 반환값 x , return생략 가능
	}
	public static int doB() {
		System.out.println("doB");
		return 10; 
	}
	public static double doC() {
		return 10.5;
	}
	public static String doD() {
		return "doD";
	}
}

File: python_work/220711/EX07.py
cnt = 100 #전역변수
i = 100

def doA():
    cnt = 0
    print('지역변수', cnt) # doA()라는 공간이 따로있다 호출하는 순간 소멸


doA()
print('전역변수', cnt)


def doB():   #지역변수
    for i in range(10):
        print(i)
    print('결과= ',i)
    


doB()
print(i) 